api/v1alpha1/securitypolicy_types.go

```diff
@@ -43,17 +43,25 @@ type SecurityPolicySpec struct {
 	TargetRef gwapiv1a2.PolicyTargetReferenceWithSectionName `json:"targetRef"`
 
 	// CORS defines the configuration for Cross-Origin Resource Sharing (CORS).
+	//
+	// +optional
 	CORS *CORS `json:"cors,omitempty"`
+
+	// JWTAuthentication defines the configuration for JSON Web Token (JWT)
+	// authentication.
+	//
+	// +optional
+	JWTAuthentication *JWTAuthentication `json:"jwtAuthentication,omitempty"`
```

Comment:
We have two choices for the naming here:
* `JWT`, shorter, but a bit vague
* `JWTAuthentiation`, clear, but a bit lenghty

Reply:
I vote for `jwt`

Reply:
+1 for `jwt`

Reply:
JWT then

Reply:
`jwt` can be used for `auth`, for converting claim to header, maybe also be used for authz of claim